specs/system_info.table

```diff
@@ -21,5 +21,6 @@ schema([
     Column("board_serial", TEXT, "Board serial number"),
     Column("computer_name", TEXT, "Friendly computer name (optional)"),
     Column("local_hostname", TEXT, "Local hostname (optional)"),
+    Column("emulated_cpu_type", TEXT, "Emulated CPU type"),
```

Comment:
Should this be a part of the extended schema for windows if it's only available on that platform?

Reply:
Search `extended_schema` to see examples of this. I'd do it first before looking at the integration test.

Reply:
Done 👍